fish_greeting.fish

```diff
@@ -1,6 +1,6 @@
 function fish_greeting -d "What's up, fish?"
     set_color $fish_color_autosuggestion
     uname -nmsr
-    uptime
+    command -v uptime > /dev/null; and uptime
```

Comment:
Let’s use `command -q -s` rather than redirecting stdout

Reply:
Thanks, that's indeed better. Commit amended.